docker_checker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/r-devel
+MAINTAINER karl.forner@quartzbio.com
```

Comment:
no, you should be. Anyway this docker will never be published so we can put anything.